### ISSUE TITLE
Seed's first 2 letters will disappear if go back with left arrow

### ISSUE THREAD

**Issue by jojoCoWorker** [iotaledger/trinity-wallet#508]
### Bug description
Seed display will disappear if go back with left arrow

### Environment Specs
- OS version: windows 7.0 pro
- Trinity version: v0.3.6 Beta

### Steps To Reproduce
1. Add a new account with existing seed
2. clic on the eye to see the letters you enter
3. Type enough letters to empty the display and at least one more
4. Go back to the left with left arrow.

### Expected behaviour
When you try to enter your seed to create a new account :
Everything is OK till the seed doesn't fill the all display :
[![Image](https://i.goopics.net/nd07K.jpg)](https://goopics.net/i/nd07K)
As soon as the seed is too long for the display, we can see the left part of the seed disappear :
[![Image](https://i.goopics.net/Xg1YX.jpg)](https://goopics.net/i/Xg1YX)
nothing strange till here...
But then, if we move the cursor back to the left (with left arrow), the seed should reappear...

### Actual behaviour
The seed reappears for a few letters but not till the beggining of the seed, so the cursor also disappear on the left part of the display :
[![Image](https://i.goopics.net/PnyY0.jpg)](https://goopics.net/i/PnyY0)
That's a bit disturbing when you want to check your seed before validation.

**Comment by delki8** [comment]
@rajivshah3 Confirmed. Also happens here.

**Comment by cvarley100** [comment]
@delki8 @jojoCoWorker Could you give some alternative instructions on how to reproduce this? We have followed the above instructions and can't repro the issue.

**Comment by jojoCoWorker** [comment]
Sorry @cvarley100 I didn't see your post... I still can see the same behaviour in v.0.4.6
I can reproduce it following the above instructions on windows 7 so maybe it's a margin problem related to the OS you use ?

**Comment by rajivshah3** [comment]
@rihardsgravis are you able to reproduce this? I think I was able to reproduce it a while ago on 0.4.6 using macOS, but it didn't occur every time. It doesn't look like this is reproducible in v0.5.0-RC2

**Comment by rajivshah3** [comment]
@jojoCoWorker does the beginning of the seed eventually show up if you press the left arrow key a few more times?

**Comment by jojoCoWorker** [comment]
@rajivshah3 no it doesn't.
If you press the left arrow Key, then you press some leters, you'll see the beginning of the seed...
I can't test on v0.5.0-RC2 (windows 7...)

**Comment by rajivshah3** [comment]
@jojoCoWorker ah, didn't realize you were using Windows 7. Is this reproducible on the latest version for Win 7?

**Comment by jojoCoWorker** [comment]
Today with v0.5.0-RC2 on windows7, it is reproductible.

**Comment by jojoCoWorker** [comment]
Same with v0.5.0 for win7, 2 letters are hidden on the left.

**Comment by rajivshah3** [comment]
Thanks for the quick reply! I'll look into this, I wonder if it's platform specific

**Comment by ggreeve** [comment]
I can confirm This is still present on WIN10 0.6.2

**Comment by rajivshah3** [comment]
@ggreeve @jojoCoWorker is this issue still occurring in 1.0.0? If so, are you able to post a screen recording?

**Comment by jojoCoWorker** [comment]
@rajivshah3 Yes still occurs in 1.0.0, same in 1.1.0-RC1, both on windows 7.

**Comment by jojoCoWorker** [comment]
@rajivshah3 
Here is my home-made screen recording : 
![20190916_115311](https://user-images.githubusercontent.com/33653820/64949593-cb011900-d879-11e9-8ecf-fa4867259e1e.gif)
1. I write my seed : ABCDEFGHIJKLMN.....XYZ9999999999999999999999 to have 81 characters
2. I come back to the beginning of the seed, can't see the first 2 letters anymore.
3. I type some 999 before the "A" to see again the beginning of the seed
4. I remove the "9"s and I can see the seed from the beginning, from A.
5. I go to the right with cursor, at least to scroll the letters we can see, then I come back to the beginning and here again I can't see the first 2 letters.

Hope it helps.
